README.md

```diff
@@ -100,12 +100,12 @@ doesn't suit your use case, perhaps one of the others will!
 
 ### druid-shell
```

Comment:
Perhaps we shoud change all mentions of *druid-shell* in (at least) this section to `druid-shell`.

Reply:
I changed this for the content of the section but I left the heading as-is because it looked weird with the `background`.